### PR TITLE
Chapter_9

### PR DESCRIPTION
**Pull request by phongvv-cyberk**

### CODE REVIEW

**Comment by phongvv-cyberk** [comment]
ready

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class SessionsController < ApplicationController
+  def new
+  end
```
Comment (thangtx88) [inline app/controllers/sessions_controller.rb:3]: def new;end

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -1,11 +1,28 @@
...
-        <li><%= link_to "Home",   '#' %></li>
-        <li><%= link_to "Help",   '#' %></li>
-        <li><%= link_to "Log in", '#' %></li>
+        <li><%= link_to t(".home"), root_path %></li>
+        <li><%= link_to t(".help"), help_path %></li>
+        <% if logged_in? %>
+          <li><%= link_to t(".user"), "#" %></li>
+          <li class="dropdown">
+            <a href="#" class="dropdown-toggle" data-toggle="dropdown">
+              Account <b class="caret"></b>
```
Comment (thangtx88) [inline app/views/layouts/_header.html.erb:12]: I18n

**Comment by thangtx88** [comment]
Commented

**Pull request merged**